clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -279,6 +294,13 @@ public void disconnected(ClientDescriptor clientDescriptor) {
 
   @Override
   public EhcacheEntityResponse invoke(ClientDescriptor clientDescriptor, EhcacheEntityMessage message) {
+    if(this.offHeapResourceIdentifiers.isEmpty()) {
```

Comment:
Decided to put this here and not just in configure to prevent ANY interaction with a useless server.